### DISCUSSION TITLE
$wire not available after switching to Vite

### DISCUSSION THREAD

**Discussion by ju5t** [livewire/livewire#5080] [1. Help] [answered]
I've switched our application from Webpack to Vite. After the migration, `$wire` suddenly doesn't work. I know that this can be related to the order in which things are loaded, but the order seems fine.

Now here's the weird part. I am not seeing this problem in Safari. It only happens in Chrome and Firefox.

Unfortunately I can't replicate this issue in a test environment. It must be something in our app, but I don't know what it is.

Has anyone seen this before?

**Comment by ju5t** [comment, marked as answer]
For me the issue was related to using `import()` vs. `import`. Vite suggests using dynamic imports with `import()`. This works fine if you're just using Alpine, but combined with Livewire it breaks. Well, it partly breaks. Because it works on Safari.

TL;DR: do not use `import()`, just use `import`.
